README.md

```diff
@@ -10,16 +10,27 @@ Adminer (formerly phpMinAdmin) is a full-featured database management tool writt
 
 ### Standalone
 
-	$ docker run --link some_database:db -p 8080:8080 timwolla/adminer
+	$ docker run --link some_database:db -p 8080:8080 adminer
 
 Then you can hit `http://localhost:8080` or `http://host-ip:8080` in your browser.
 
 ### FastCGI
 
 If you are already running a FastCGI capable web server you might prefer running adminer via FastCGI:
 	
-	$ docker run --link some_database:db -p 9000:9000 timwolla/adminer:fastcgi
+	$ docker run --link some_database:db -p 9000:9000 adminer:fastcgi
 	
 Then point your web server to port 9000 of the container.
 	
 Note: This exposes the FastCGI socket to the Internet. Make sure to add proper firewall rules to prevent a direct access.
+
+### Choosing a design
+
+The image contains all the designs that are available in the source package of adminer.
+You can find an overview here: https://github.com/vrana/adminer/tree/master/designs.
+
+To activate a design you can pass its name in `ADMINER_DESIGN`:
+
+```
```

Comment:
Please use the four-space-indentation style for code blocks. This is a requirement for official images.